clvm_tools/cmds.py

```diff
@@ -205,14 +205,15 @@ def launch_tool(args, tool_name, default_stage=0):
     try:
         output = "(didn't finish)"
 
-        if args.backend == "rust" or (serialize_and_run_program and args.backend != "python"):
+        use_rust = (tool_name != "run") and not args.table and not args.stage and (args.backend == "rust" or (serialize_and_run_program and args.backend != "python"))
```

Comment:
I bet `black` doesn't like this. (I'm not crazy about it either, but I know it's temporary).

Reply:
I don't like it either, and I wrote it!